infra/modules/azure_role_assignments/variables.tf

```diff
@@ -128,3 +128,17 @@ variable "apim" {
 
   default = []
 }
+
+variable "service_bus" {
+  description = "A list of role assignments for Azure Service Bus, specifying the namespace, resource group, role, and queue names or subscriptions with their topics."
```

Comment:
suggestion: I'd try to be more explicit. For example:

1. what's the `description` field about ?
2. what do keys and values in the `subscriptions` field represent?
3. which `roles` are supported?

Reply:
What about renaming the field to something like 'role_description'? It's not immediate for the casual reader to correlate `description` with the role since it appears inside the `service_bus` block.

Reply:
We have the same field name in all variables of the module, I suggest to do not breaking the coherence for a single resource type